src/main/java/org/craftercms/studio/impl/v1/service/site/SiteServiceImpl.java

```diff
@@ -869,10 +876,10 @@ private void createSiteCloneRemote(String siteId, String description, String rem
         }
     }
 
-    private void createSitePushToRemote(String siteId, String description, String blueprintName, String remoteName,
-                                        String remoteUrl, String remoteBranch, String authenticationType,
-                                        String remoteUsername, String remotePassword, String remoteToken,
-                                        String remotePrivateKey)
+    private void createSitePushToRemote(String siteId, String sandboxBranch, String description, String blueprintName,
+                                        String remoteName, String remoteUrl, String remoteBranch,
```

Comment:
What is `remoteBranch` parameter used for?

Reply:
`remoteBranch` is the name of the branch from the remote repository when creating a site from the remote repository